### DISCUSSION TITLE
What is the maximum frequency of sending goto goal commands to crazyflie?

### DISCUSSION THREAD

**Discussion by pthinesh** [USC-ACTLab/crazyswarm#466] [Q&A]
Hello Everyone!

I am trying to run a swarm algorithm by computing the goal position in the computer and sending `goTo` command in every 0.1 second duration (10 Hz) to travel less than 1 cm absolute point to point distance. When I send these commands to the drones, it is becoming unstable and collapsing.
But when I send commands in every 4 seconds (0.25 Hz) to travel 10 cm distance, drones moves smoothly.

I would like to know what is the maximum and minimum distance that can be traveled between two `goTo` commands and what is the maximum frequency of sending any commands (e.g `cmdFullState`, `cmdVelocityWorld`, `cmdVel`) to crazyflie to have a smooth movement?

Could someone advise me on this?

Thanks

**Comment by whoenig** [comment]
I have seen those issues as well. There is no maximum frequency (the radio will limit the bandwidth). The goTo command uses trajectory optimization internally on-board the drone, which can be numerically unstable for small dt. In such a case, it is better to use one of the commands (e.g., `cmdFullState`) instead.
